sicario.c

```diff
@@ -1,31 +1,19 @@
+/*sudo apt-get install ncurses-dev*/
+/*gcc <file-name>.c -lncurses -o <executable-name>*/
 #include <stdio.h>
 #include <stdlib.h>
 #include <unistd.h>
 #include <curses.h>
 #include <stdbool.h>
 
+#define KEY_ESC 27
+#define L_KEY_ENTER 10
 
-/*sudo apt-get install ncurses-dev*/
-/*gcc a.c -lncurses*/
-
-
-
-//Macros para reconhecer inputs
-#define SETAS -32
-#define ESQUERDA 75
-#define DIREITA 77
-#define CIMA 72
-#define BAIXO 80
-#define ESC 27
-#define ENTER 10
-
-//Macros para tamanho do tabuleiro
 #define TABULEIRO_W 70
```

Comment:
Thanks for rename a lot of variables and methods for english. Could you please rename those too? When I say "those", I mean lines 12, 13, 17, 18, 19, 30, 46, 60, 64, 70, 75, 156, 157, 161, 162, 164, 167, 168, 170.

Reply:
Done.